teach.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: inner-page-no-masthead
+title: Teach
+permalink: /teach
+includeTOC: true
+---
+
+# Scala is a great tool to teach programming
```

Comment:
Just a suggestion but maybe you could add somewhere that Scala was designed by Martin Odersky, professor of Computer Science, and that he has used this language to teach programming since 20??